qrcode/tests/test_qrcode_pypng.py

```diff
@@ -29,7 +27,7 @@ def test_render_pypng_to_str():
     assert isinstance(img.get_image(), png.Writer)
 
     mock_open = mock.mock_open()
-    with mock.patch("qrcode.image.pure.open", mock_open, create=True):
+    with mock.patch("pathlib.Path.open", mock_open, create=True):
         img.save("test_file.png")
-    mock_open.assert_called_once_with("test_file.png", "wb")
+    mock_open.assert_called_once_with("wb")
     mock_open("test_file.png", "wb").write.assert_called()
```

Comment:
Hmm – this could just use the Pytest `tmp_path` factory and actually write to disk instead of mocking IO, to test that everything does really work.

Reply:
You're 100% right. I don't see the benefit of this convoluted setup either. I created #409 to handle that. 

https://github.com/lincolnloop/python-qrcode/blob/c38fb6f5944b5d2d0c754b6553df5d4e142508fe/qrcode/tests/test_module.py#L30 does it already that way.